lib/local-storage-adapter.js

```diff
@@ -57,7 +54,7 @@ function createLocalStorageAdapter (queueName) {
     if (dirtyCache) {
       queueCache = json.parse(adapter.load(queueKey) || '[]')
       dirtyCache = false
-      flushOnDefer()
+      setTimeout(flush, 0)
```

Comment:
Hold up. Doesn't this behave differently. If you call debounce multiple times - it collapses those calls into 1. Whereas now, setTimeout will call each one individually. Given it's 0 timeout - probably not a big deal.